azure-pipelines.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+  branches:
+    include:
+      - ni/current
+
+pool:
+  name: DevCentral-Docker
+
+steps:
+- task: Bash@3
+  displayName: Generate version number
+  inputs:
+    targetType: 'inline'
+    script: |
+      TagNumber=$(git describe --abbrev=0 --tags)
+      ShortHash=$(git rev-parse --short $(Build.SourceVersion))
+      ImageVersion="$TagNumber-$ShortHash-ni"
```

Comment:
Should we add a comment with an example of what an image version will look like? I assume it'll be something like "8.3.6-xyz_short_hash-ni"

Reply:
That is correct. I'll add a comment.